x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -304,7 +249,34 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		if err != nil {
 			utils.LogLavaEvent(ctx, logger, types.UnresponsiveProviderUnstakeFailedEventName, map[string]string{"err:": err.Error()}, "Error Unresponsive Providers could not unstake")
 		}
+
+		// track the provider's CU (after QoS influence)
+		proj, err := k.projectsKeeper.GetProjectForDeveloper(ctx, clientAddr.String(), uint64(relay.Epoch))
```

Comment:
this was changed in Part 1, can you rebase/merge?

Reply:
done. see https://github.com/lavanet/lava/commit/df5f0dded954446ccdfec191f5e2b4014070b555